src/tester.ts

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+import 'colors';
+const test = require('./main');
+const yargs = require('yargs');
+
+let path: string = process.cwd();
+
+const argv = yargs
+  .command('path', 'The path to look for test files in', {
+    path: {
+      description: 'the path',
+      alias: 'p',
+      type: 'string',
+    },
+  })
+  .help()
+  .alias('help', 'h').argv;
+
+if (argv.path) {
+  path = argv.path;
+}
+
+if (!path.endsWith('/')) {
+  path += '/';
+}
+
+(async () => {
+  console.log('Searching for files in: '.yellow + path);
+  const files: string[] = await test.findFiles(path);
+
+  if (files && files.length > 0) {
+    console.log('Found files: '.green + files);
+
+    files.forEach((file: string) => {
+      console.log('Loading file: '.green + file);
+      require(path + file);
```

Comment:
NOTE: this is a special case where using `require` makes more sense.

Reply:
Noted, thanks 👍